docs/contributing/how-to-release.md

```diff
@@ -2,14 +2,19 @@
 
 Creating a new release requires a few things before getting started:
 
-* Linux or OSX with 7z installed.
-* Windows with HTML Help Workshop installed.
-* Doxygen 1.8.8, and GraphViz installed on both machines.
+* Linux or OSX.
+* Windows with HTML Help Workshop, and Inno Setup installed.
+* 7-Zip, Doxygen 1.8.8, and GraphViz installed on both machines.
```

Comment:
I tried so hard (spent a good couple hours) to find a way to avoid the need for 7-Zip on Windows by trying to use the shell ZIP functionality from a VB script like the approach [found here](http://stackoverflow.com/a/21709923/450142), but eventually gave up.